src/frontend/src/flows/confirmRedirect.ts

```diff
@@ -1,7 +1,7 @@
 import { html, render } from "lit-html";
 import { FrontendHostname } from "../../generated/internet_identity_types";
 
-const pageContent = (hostName: string, principal: string) => html`
+const pageContent = (hostName: string) => html`
```

Comment:
Here I am assuming that it was decided to remove the pseudonym from the redirect confirmation, right?

Reply:
Yes, we decided to not roll that change into the terminology changes, because we're still debating whether it's a good idea.